lib/smart_answer_flows/student-finance-calculator.rb

```diff
@@ -82,7 +82,8 @@ def define
           calculator.maintenance_loan_amount
         end
```

Comment:
I don't think you need calculate blocks for `maintenance_grant_amount` and `maintenance_loan_amount` anymore. You should be able to access the calculator directly where ever those amounts are needed.